lib/chain_sync.ex

```diff
@@ -289,7 +290,6 @@ defmodule Xander.ChainSync do
 
         case CSResponse.decode(combined_payload) do
           {:ok, %AwaitReply{}} ->
-            :ok = setopts_lib(client).setopts(socket, active: :once)
```

Comment:
The reason for setting the socket to active here and then below is that we need it to be **after** invoking the `read_remaining_payload` function. Notice that line 277 manually calls `client.recv(socket, recv_payload_length, @recv_timeout)`

I'm running this branch and don't see new blocks displayed so I suspect the changes break the current functionality.